privcount/traffic_model.py

```diff
@@ -276,8 +281,10 @@ def _get_inter_packet_delays(self, strm_start_ts, byte_events):
             delay = max(long(0), long(micros))
 
             # create a packet delay for each packet
-            while bw_bytes > 0:
+            split_packet_count = 0
+            while bw_bytes > 0 and split_packet_count < TrafficModel.MAX_SPLIT_PACKET_COUNT:
```

Comment:
(If we ever find that bandwidths are regularly going over 150 KB, we will have to re-write the algorithm so it runs in a reasonable time on these inputs.)